cli/tests/conftest.py

```diff
@@ -198,7 +205,9 @@ def example_messages_multiple_projects() -> list[Message]:
 @pytest.fixture()
 def example_messages_multiple_projects_df(example_messages_multiple_projects) -> pd.DataFrame:
     """Test input data."""
-    return pd.DataFrame.from_records([vars(im) for im in example_messages_multiple_projects])
+    messages_df = pd.DataFrame.from_records([vars(im) for im in example_messages_multiple_projects])
+    messages_df["pseudo_patient_id"] = 0
```

Comment:
same here for not setting `pseudo_patient_id` to have a value

Reply:
[d920af6](https://github.com/UCLH-Foundry/PIXL/pull/484/commits/d920af6f3c423992677f33c5f774a7b01a7e24c8)